components/telemetry-operator/controller/tracepipeline/config_builder.go

```diff
@@ -0,0 +1,223 @@
+package tracepipeline
+
+import (
+	"fmt"
+
+	"github.com/kyma-project/kyma/components/telemetry-operator/apis/telemetry/v1alpha1"
+)
+
+type TlsConfig struct {
+	Insecure bool `yaml:"insecure"`
+}
+
+type OtlpExporterConfig struct {
```

Comment:
As discussed, let's use all caps for acronyms: `OTLP` and `TLS`

Reply:
Done